.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,381 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Build PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  LANG: en_US
```

Comment:
this has been introduced as workaround for a quarkus issue fixed by https://github.com/quarkusio/quarkus/pull/6681on quarkus main, since it is a workaround and should be removed I would leave it as the quarkus guys defined it